# Loads a recognition model for the face recognizer from disc or database
#
# goal message
string[] labels		# a list of labels of persons that shall be recognized
---
# result message
---
# feedback message
